src/commands/profile/export/collect-ids.ts

```diff
@@ -5,12 +5,12 @@ import { Entity, IExportHolder } from "./types";
 
```

Comment:
@vitorfdl some unit tests are not passing.
The command `npm run build` also having some errors.